File: ActionableHealth/DataModels/MessagingManager/Messages/Messages+CoreDataProperties.m
//
//  Messages+CoreDataProperties.m
//  
//
//  Created by Vidhan Nandi on 02/01/17.
//
//

#import "Messages+CoreDataProperties.h"

@implementation Messages (CoreDataProperties)

+ (NSFetchRequest<Messages *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Messages"];
}

@dynamic messageId;
@dynamic key;
@dynamic message;
@dynamic type;
@dynamic person;

@end

File: ActionableHealth/ActionableHealth/DataModels/MessagingManager/Messages/Messages+CoreDataProperties.swift
//
//  Messages+CoreDataProperties.swift
//  
//
//  Created by Vidhan Nandi on 14/02/17.
//
//

import Foundation
import CoreData


extension Messages {

    @nonobjc open override class func fetchRequest() -> NSFetchRequest<NSFetchRequestResult> {
        return NSFetchRequest(entityName: "Messages");
    }

    @NSManaged public var key: String?
    @NSManaged public var message: String?
    @NSManaged public var messageId: String?
    @NSManaged public var priority: String?
    @NSManaged public var status: String?
    @NSManaged public var timestamp: NSNumber?
    @NSManaged public var type: String?
    @NSManaged public var msgDate: Date?
    @NSManaged public var lastMessagePerson: Person?
    @NSManaged public var person: Person?

}
